config/core/resources/eventtype.yaml

```diff
@@ -22,6 +22,132 @@ metadata:
 spec:
   group: eventing.knative.dev
   versions:
+  - name: v1beta3
+    served: false
+    storage: false
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: 'EventType represents a type of event that can be consumed from a resource.'
+        properties:
+          spec:
+            description: 'Spec defines the desired state of the EventType.'
+            type: object
+            properties:
+              reference:
+                description: Reference a resource. For example, Broker.
+                type: object
+                properties:
+                  apiVersion:
+                    description: API version of the referent.
+                    type: string
+                  kind:
+                    description: 'Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+                    type: string
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                    type: string
+                  namespace:
+                    description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/ This is an optional field, it gets defaulted to the object holding it if left out.'
+                    type: string
+              description:
+                description: 'Description is an optional field used to describe the
+                    EventType, in any meaningful way.'
+                type: string
+              attributes:
+                description: "CloudEvent attribute and extension attributes."
+                type: array 
+                items:
+                  type: object
+                  required:
+                    - name
```

Comment:
isn't `value` also required ?

Reply:
Not currently, no. Currently you can just add a `name` and whether or not that attribute is required without specifying what value it may have. I think of this as a way of saying "the event may have attribute `xyz`". But, I could understand making this field required too. WDYT we should do @pierDipi ?

Reply:
yes, that is exactly what I was thinking too. IMO this is useful for "extensions",

the following is IMO valid:
```
    - name: myextension
      required: false
```

while some attribute names are requied to have a value (e.g. all the CE "required" attrs)